.github/workflows/draftrelease.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - master
```

Comment:
We probably will want to remove this, as it will trigger draft releases when things are merged into "master".

Reply:
Ahh yes, good call!